core-bundle/src/Controller/BackendController.php

```diff
@@ -137,23 +118,6 @@ public function helpAction(): Response
         return $controller->run();
     }
 
-    /**
-     * @Route("/page", name="contao_backend_page")
-     *
-     * @deprecated Deprecated since Contao 4.13, to be removed in Contao 5.0.
-     *             Use the picker instead.
-     */
-    public function pageAction(): Response
```

Comment:
this should be reverted and is already in another PR

Reply:
Reverted in 576e7b51a52d5c13133b6445fc85e491c1c91a0c.